packages/paste-codemods/tools/__tests__/tools.spec.js

```diff
@@ -39,12 +39,15 @@ describe('generatePackageExportsMap', () => {
       DisclosurePrimitive: '@twilio-paste/core/disclosure-primitive',
       DisclosurePrimitiveContent: '@twilio-paste/core/disclosure-primitive',
       Stack: '@twilio-paste/core/stack',
+      StyledBox: '@twilio-paste/core/box',
+      getCustomElementStyles: '@twilio-paste/core/box',
       getStackChildMargins: '@twilio-paste/core/stack',
       getStackDisplay: '@twilio-paste/core/stack',
       getStackStyles: '@twilio-paste/core/stack',
       safelySpreadBoxProps: '@twilio-paste/core/box',
       useComboboxPrimitive: '@twilio-paste/core/combobox-primitive',
       useDisclosurePrimitiveState: '@twilio-paste/core/disclosure-primitive',
+      useMultiSelectPrimitive: '@twilio-paste/core/combobox-primitive',
```

Comment:
This test wasn't being run in the old config. It does now that we have a single jest config that covers all test source formats, across the entire repo.